spec/index.html

```diff
@@ -199,43 +209,59 @@ <h3>Example</h3>
     </section>
     <section id="general-comments">
     <h2>Transmission issues using CSV and TSV Formats</h2>
-    <p>The SPARQL results formats described here confirm to the formal specifications of the relevant formats, Comma Separated values (CSV) [<cite><a class="bibref" rel="biblioentry" href=
-    "#bib-RFC4180">RFC4180</a></cite>] and Tab Separated Value (TSV) [<cite><a class="bibref" rel="biblioentry" href="#bib-IANA-TSV">IANA-TSV</a></cite>].</p>
-    <p>Systems providing these formats should note that the content types for CSV is <code>text/csv</code> and for TSV <code>text/tab-separated-values</code>. Being <code>text/*</code>, the default character set
-    is US-ASCII. The <code>charset</code> parameter <em class="rfc2119" title="should">should</em> be used in conjunction with SPARQL Results; UTF-8 is recommended: <code>text/csv; charset=utf-8</code> and
-    <code>text/tab-separated-values; charset=utf-8</code>.</p>
-    <p>The end-of-line in CSV is <code>CRLF</code> i.e. Unicode codepoints 13 (0x0D) and 10 (0x0A).</p>
-    <p>The end-of-line in TSV is <code>EOL</code> i.e. Unicode codepoint 10 (0x0A).</p>
-    <p>Applications reading these formats are advised to cope with both CRLF and LF as end of line markers and not rely on conformance to the formal specifications.</p>
+    <p>The SPARQL result formats described here conform to the formal specifications of the relevant formats, 
+      Comma Separated values (CSV) [<cite><a class="bibref" rel="biblioentry" href="#bib-RFC4180">RFC4180</a></cite>] 
+      and Tab Separated Value (TSV) [<cite><a class="bibref" rel="biblioentry" href="#bib-IANA-TSV">IANA-TSV</a></cite>].</p>
+    <p>Systems providing these formats should note that the content types are <code>text/csv</code> for CSV and 
+      <code>text/tab-separated-values</code> for TSV. Being <code>text/*</code>, the default character set
+      is US-ASCII. The <code>charset</code> parameter <em class="rfc2119" title="should">should</em> be used in
+      conjunction with SPARQL Results; UTF-8 is recommended; giving us <code>text/csv; charset=utf-8</code> and
+      <code>text/tab-separated-values; charset=utf-8</code>.</p>
+    <p>The end-of-line in CSV is <code>CRLF</code>, i.e., Unicode codepoints 13 (<code>0x0D</code>) and 10 (<code>0x0A</code>).</p>
+    <p>The end-of-line in TSV is <code>EOL</code>, i.e., Unicode codepoint 10 (<code>0x0A</code>).</p>
```

Comment:
Just noted, [`codepoint` should be `code point`](https://unicode.org/glossary/#code_point).
```suggestion
    <p>The end-of-line in CSV is <code>CRLF</code>, i.e., Unicode code points <code>U+0013</code> (<code>0x0D</code>) and <code>U+0010</code> (<code>0x0A</code>).</p>
    <p>The end-of-line in TSV is <code>EOL</code>, i.e., Unicode code point <code>U+0010</code> (<code>0x0A</code>).</p>
```